### ISSUE TITLE
Window has unexpectedly exited .. starting profile

### ISSUE THREAD

**Issue by gvc08** [kolton/d2bot-with-kolbot#981]
It hasn’t been overly bad, roughly 15 % of runs this will happen.. the crash info for the past 3..

Crash info: script : worldstone area: worldstone keep level 2

Crash info: script: Worldstone area: Worldstone keep level 3

Crash info: script: diablo area: the pandemonium fortress

**Comment by gvc08** [comment]
Is there anything I can do about these, or is this normal? I’m running two bots right now for the first time.. and it crashed 3 times in the first 6 runs between both chars.. using same script for Pally.. I have the latest version.. downloaded after my prev version kept having error in syntax

**Comment by gvc08** [comment]
Also, thank you Noah.. you have been super helpful.. sorry about always asking questions

**Comment by SirYac** [comment]
Same issue here. 2 bots at the same time and crashes happen to about 50% of all runs. 
I use latest version of kolbot and run everything as it should (administrator, win 7)

`14:15:43 (XXXX2) Window has unexpectedly exited... starting profile
14:15:47 (XXXX2) Crash Info: Script: Worldstone Area: Worldstone Keep Level 3
14:27:54 (XXXX) Window has unexpectedly exited... starting profile
14:27:58 (XXXX) Crash Info: Script: Worldstone Area: Worldstone Keep Level 1
14:29:50 (XXXX) Window has unexpectedly exited... starting profile
14:29:53 (XXXX) Crash Info: Script: no entry Area: Pit Level 1`

I believe it happen most often at act V locations, especially Worldstone/Throne of destrucion.

Please help!

**Comment by semirotta** [comment]
Yep I've noticed act 4 and 5 crashesh all the time, I barely get my runs through those acts at all.

**Comment by expl0** [comment]
I'm having the same problem, but only on my sorc. Alot of crashes in WSK level 1-3 and in Throne. Weird thing is that my pala runs perfectly fine without any crashes.

**Comment by semirotta** [comment]
Window has unexpectedly exited... starting profile
Crash Info: Script: Diablo Area: Chaos Sanctuary
Window has unexpectedly exited... starting profile
Crash Info: Script: Diablo Area: Chaos Sanctuary

This just happened twice in row. I use sorcerer aswell.

**Comment by expl0** [comment]
This problem only seems to occur in the evenings. My sorc bot crashed 3 times in a row at nihlathak just now. During the day everything is fine. Could it be that at these times the servers are less stable? Even if, pala bot is running perfectly...

**Comment by semirotta** [comment]
I highly doubt it has anything to do with the time of day. For example I had 48 hours up running my sorc, I only crashed few times and chickened few times. Then there are times when in that similar 48 hour time period it crashes all the time or life chickens all the time. It's weird.

**Comment by sangami70** [comment]
> It hasn’t been overly bad, roughly 15 % of runs this will happen.. the crash info for the past 3..
> 
> Crash info: script : worldstone area: worldstone keep level 2
> 
> Crash info: script: Worldstone area: Worldstone keep level 3
> 
> Crash info: script: diablo area: the pandemonium fortress

09:20:24 (LOKImf1) Window has unexpectedly exited... starting profile
09:20:27 (LOKImf1) Crash Info: Script: Worldstone Area: Worldstone Keep Level 3

me too
